ocaml/xapi/record_util.ml

```diff
@@ -91,7 +91,7 @@ let vm_bios_key_to_string: API.vm_bios_string_keys -> string = function
   | `system_version -> "system-version"
   | `system_serial_number -> "system-serial-number"
   | `enclosure_asset_tag -> "enclosure-asset-tag"
-  | `unknown -> "unknown"
+  | `unknown -> failwith "unknown bios string key"
```

Comment:
Shouldn't this be an API error (invalid_value)?

Reply:
Updated, Thanks.